acceptance/tests/custom_facts/using_win32ole_should_not_hang.rb

```diff
@@ -0,0 +1,29 @@
+test_name 'C92060: Custom facts should not hang Facter when using win32ole' do

```

Comment:
This file did not properly replace newlines and adds whitespace errors. The following warning is evident when applying this patch:
```
warning: 29 lines add whitespace errors
```